addons/account/models/uom_uom.py

```diff
@@ -19,27 +19,13 @@ def _get_unece_code(self):
         https://unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex2e.pdf"""
         mapping = {
             'uom.product_uom_unit': 'C62',
-            'uom.product_uom_dozen': 'DZN',
+            'uom.product_uom_pack_6': 'HD',
             'uom.product_uom_kgm': 'KGM',
             'uom.product_uom_gram': 'GRM',
             'uom.product_uom_day': 'DAY',
             'uom.product_uom_hour': 'HUR',
-            'uom.product_uom_ton': 'TNE',
             'uom.product_uom_meter': 'MTR',
-            'uom.product_uom_km': 'KMT',
-            'uom.product_uom_cm': 'CMT',
             'uom.product_uom_litre': 'LTR',
-            'uom.product_uom_lb': 'LBR',
-            'uom.product_uom_oz': 'ONZ',
-            'uom.product_uom_inch': 'INH',
-            'uom.product_uom_foot': 'FOT',
-            'uom.product_uom_mile': 'SMI',
-            'uom.product_uom_floz': 'OZA',
-            'uom.product_uom_qt': 'QT',
-            'uom.product_uom_gal': 'GLL',
```

Comment:
Well I mostly lost the battle. We will keep oz and gal (archived by default, unarchive when US localization is installed). For after freeze, we need to go through all of these localization/accounting codes deletions carefully because there are a couple mistakes: e.g. ton (TNE) is still being used as well as lb (LBR)